modules_basic/timestamp.js

```diff
@@ -1,28 +1,47 @@
-var h = require('hyperscript')
+var fs = require('fs')
+var Path = require('path')
+var h = require('../h')
 var human = require('human-time')
 
 exports.needs = {}
 
-exports.gives = 'message_meta'
+exports.gives = {
+  timestamp: true,
+  mcss: true
+}
 
 exports.create = function () {
-
-  function updateTimestampEl(el) {
-    el.firstChild.nodeValue = human(new Date(el.timestamp))
-    return el
-  }
-
   setInterval(function () {
-    var els = [].slice.call(document.querySelectorAll('.timestamp'))
+    var els = [].slice.call(document.querySelectorAll('.Timestamp'))
```

Comment:
Yeah, I really dislike this. But mostly just because it is not very mutanty - looks a lot more like jquery.

There are much better ways to do this. For example with a hook. Could still be depjected.